tests/datasets/test_replays_processor.py

```diff
@@ -162,7 +162,9 @@ def serialize(self) -> Mapping[Any, Any]:
 
         return {
             "type": "replay_event",
-            "start_time": datetime.now().timestamp(),
+            "start_time": self.timestamp
+            if isinstance(self.timestamp, (float, int))
+            else datetime.now().timestamp(),
```

Comment:
Previously, the test `serialize()` function was just ignoring whatever was passed as a `timestamp` and sending it's own timestamp value.
Actually using what was passed as `timestamp` caused the tests involving bad timestamps to fail, so this keeps the behaviour of 'if timestamp is bad just chuck it & send a real timestamp'.